src/services/user.js

```diff
@@ -42,8 +42,19 @@ const getUser = () => {
     .then(utils.handleResponse);
 };
 
+const getUserByEmail = (email) => {
+  const requestOptions = {
+    method: 'GET',
+    headers: { ...utils.authHeader(), 'Content-Type': 'application/json' },
```

Comment:
GET requests don't need the `Content-Type` header.
```suggestion
    headers: { ...utils.authHeader() },
```